test/wpt/README.md

```diff
@@ -91,7 +91,11 @@ add this to `test/wpt/status/url.json`:
 
 ```json
   "url-searchparams.any.js": {
-    "fail": "explain why the test fails, ideally with links"
+    "fail": {
+      "expected": [
+        "test name"
```

Comment:
Can you make the descriptions here more descriptive? Something like:

```suggestion
        "test name in the WPT test case, e.g. second argument passed to test()"
```

Reply:
done.